### PR TITLE
Add prismic migration guide

### PR DESCRIPTION
**Pull request by paul-vd**

### CODE REVIEW

File: source/docs/appendices/migration-guides.md [hunk 1]
```diff
@@ -255,30 +281,31 @@ export const DesktopLeftMenu = injectIntl(
...
+
 </details>
 
 ### New style sheets
 
 If you overrode `theme/components/_components.scss` you need to add the following line to it
 
 ```scss
-@import "~theme/components/molecules/LoadingOverlay/LoadingOverlay";
-@import "~theme/components/molecules/SelectMenu/SelectMenu";
```
Comment (real34) [inline source/docs/appendices/migration-guides.md:266]: This should be reverted.

Our default Prettier rule use `"`, so we should document with double quotes too.

**Review by real34 — CHANGES_REQUESTED** [review]
The migration guide part is good to me.

Everything unrelated seems not, so it should be removed

File: source/docs/appendices/migration-guides.md (hunk 1 above)
Comment (paul-vd) [reply source/docs/appendices/migration-guides.md:266]: Hmm it's strange, where is the prettier config?

**Review by abailleulfc — APPROVED** [review]

File: source/docs/appendices/migration-guides.md [hunk 2]
```diff
@@ -17,12 +17,36 @@ If you are using Magento2, version 2.6.0 of `front-commerce/magento2-module` is
...
++ FRONT_COMMERCE_PRISMIC_REPOSITORY_NAME=my-repo
+```
+
+#### `@prismicio/client` has been updated to v6
+
+See https://prismic.io/docs/technologies/prismic-client-v6-migration-guide for the complete migration guide
+
+#### `prismic-dom` replaced with `@prismicio/helpers`
+
+See https://prismic.io/docs/technologies/prismic-helpers-v2-migration-guide for the complete migration guide
```
[the remark this answers is not in the document]
Comment (paul-vd) [reply source/docs/appendices/migration-guides.md:42]: It should not, but I included it incase they did do some overides.

File: source/docs/appendices/migration-guides.md [hunk 3]
```diff
@@ -17,12 +17,36 @@ If you are using Magento2, version 2.6.0 of `front-commerce/magento2-module` is
...
+
+```diff
+# .env
+- FRONT_COMMERCE_PRISMIC_URL=https://my-repo.prismic.io
++ FRONT_COMMERCE_PRISMIC_REPOSITORY_NAME=my-repo
+```
+
+#### `@prismicio/client` has been updated to v6
+
+See https://prismic.io/docs/technologies/prismic-client-v6-migration-guide for the complete migration guide
```
[the remark this answers is not in the document]
Comment (paul-vd) [reply source/docs/appendices/migration-guides.md:38]: I added the relevant changes to predicates

File: source/docs/prismic/preview.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+You need to add these domains to enable the scripts and the iframe to be loaded in your website.
+
+```diff
+// src/config/website.js
+
+{
+  contentSecurityPolicy: {
+    directives: {
+-      scriptSrc: [],
++      scriptSrc: ['static.cdn.prismic.io', 'prismic.io', 'https://html2canvas.hertzen.com/dist/html2canvas.min.js'],
```
Comment (real34) [inline source/docs/prismic/preview.md:37]: Why is https://html2canvas.hertzen.com/dist/html2canvas.min.js needed?

**Review by real34 — APPROVED** [review]

File: source/docs/prismic/preview.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+-      frameSrc: [],
++      frameSrc: ['*.prismic.io'],
+    },
+  },
+}
+```
+
+- `static.cdn.prismic.io` is the prismic cdn where the script is loaded from
+- `https://html2canvas.hertzen.com/dist/html2canvas.min.js` is a third-party script which allows a sharable prismic link
```
Comment (paul-vd) [inline source/docs/prismic/preview.md:46]: This script gets injected in by the prismic toolbar script for the preview, which allows for the canvas to add the sharable link.

image

**Pull request merged**